File: ModelInterfacesClassLibrary/Phonebook/LprDivision02/LprDivision02ViewPage.interface.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using Newtonsoft.Json;
using Newtonsoft.Json.Serialization;
using ModelInterfacesClassLibrary.Phonebook.LprDivision02;

namespace ModelInterfacesClassLibrary.Phonebook.LprDivision02 {
    public interface ILprDivision02ViewPage
    {
        [JsonProperty(PropertyName = "page")]
        int page { get; set; }

        [JsonProperty(PropertyName = "pagesize")]
        int pagesize { get; set; }

        [JsonProperty(PropertyName = "pagecount")]
        int pagecount { get; set; }

        [JsonProperty(PropertyName = "total")]
        int total { get; set; }

        [JsonProperty(PropertyName = "items")]
        IList<ILprDivision02View> items { get; set; }
    }
}

File: AngularMatPhonebookSolution/projects/phonebook-interfaces/src/lib/lpr-division02/interfaces/lpr-division02-view-page.interface.ts
import { ILprDivision02View } from './lpr-division02-view.interface';

export interface ILprDivision02ViewPage {
        page : number; // int
        pagesize : number; // int
        pagecount : number; // int
        total : number; // int
        items ?: ILprDivision02View[];
}


